pubtools/pulplib/_impl/fake/client.py

```diff
@@ -128,3 +177,6 @@ def _next_task_id(self):
         with self._lock:
             next_raw_id = self._uuidgen.randint(0, 2 ** 128)
         return str(uuid.UUID(int=next_raw_id))
+
+    def _request_id(self):
```

Comment:
It's a little inconsistent with the naming of next_task_id.  Would this make more sense?

```suggestion
    def _next_upload_id(self):
```